server/server.js

```diff
@@ -13,4 +23,17 @@ app.get('/api/careers', (request, response) => {
   });
 });
 
+app.get('/add-career', (request, response) => {
```

Comment:
it might be nice to have every url related to the db have be /api/something to make it very obviously separate from the rest of the app?

Reply:
I like the idea but I think when we give names to paths we should consider the user, that can see the url. How about `career/`, so this one would be `career/add-career`

Reply:
Also, the idea for dash in url was Finn's, that thought it would be more friendly to the user instead of `aadCareer`. I link the idea :1st_place_medal:

Reply:
yeah sure fair enough :+1: